moveit_core/robot_model/include/moveit/robot_model/robot_model.h

```diff
@@ -163,6 +163,12 @@ class RobotModel
     return active_joint_model_vector_const_;
   }
 
+  /** \brief Get the array of active joint names, in the order they appear in the robot state. */
```

Comment:
```suggestion
  /** \brief Get the array of active joint names, in the order that they appear in the robot state. */
```

Reply:
I know this is wrong in the rest of the file, but let's not perpetuate mistakes.

Reply:
I'd leave it as is and [literature backs me up](https://books.google.com/ngrams/graph?content=in+the+order+they+appear%2Cthe+order+that+they+appear%2Cin+the+order+of+appearance&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cin%20the%20order%20they%20appear%3B%2Cc0%3B.t1%3B%2Cthe%20order%20that%20they%20appear%3B%2Cc0%3B.t1%3B%2Cin%20the%20order%20of%20appearance%3B%2Cc0#t1%3B%2Cin%20the%20order%20they%20appear%3B%2Cc0%3B.t1%3B%2Cthe%20order%20that%20they%20appear%3B%2Cc0%3B.t1%3B%2Cin%20the%20order%20of%20appearance%3B%2Cc0) :-)

Reply:
I also think it's fine as is